dev-libs/libsecp256k1/libsecp256k1-0.2.0-r2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools multilib-minimal
```

Comment:
Ugh, I'm sorry for making a last minute-comment, but why multilib? Are there 32-bit binaries we know that need this?

If not, this is a similar issue to static-libs. We should avoid it unless something actually needs it. We don't really want the multilib depgraph to grow at all, although it's less of an issue here as it has no runtime deps.

Reply:
Portage makes it easy to build an application using a different ABI than the system default, in which case dependencies need to be available in that ABI. For instance, I could configure my system to build `net-p2p/bitcoind` using the x32 ABI to reduce pointer overhead, in which case I'd need an x32 build of libsecp256k1.

Isn't Gentoo about maximizing choice for the user? Why would you want to say "no, you can't have this library in multiple ABI flavors" when it's so trivially easy to allow it?

Reply:
But you can't do that, because bitcoind isn't multilib capable, and our multilib setup is oriented around libraries, not building an executable for two different ABIs.

Reply:
(I do sympathise -- what kind of happened is that the multilib you dream of was actually _almost_ a thing, but the main people behind it disappeared and never bothered trying to formalise its behaviour, and then it died out in favour of the current multilib approach.)

Reply:
Hmm, I've definitely configured Portage to build applications for a non-default ABI before. I recall that the only real gotcha was that it wouldn't automatically require dependencies to have the necessary ABI USE flag enabled, so I had to enable them myself. Not exactly ideal but doable. What you say makes sense: the support is like 90% of the way there.

Okay, I'll drop the multilib support.